Makefile

```diff
@@ -130,3 +159,7 @@ fuzzy: ## Find fuzzy strings
 .PHONY: rm_cpython
 rm_cpython: ## Remove cloned cpython repo
 	rm -rf $(CPYTHON_CLONE)
+
+# This allows us to accept extra arguments (by doing nothing when we get a job that doesn't match, rather than throwing an error)
+%:
+	@:
```

Comment:
**question:** Curious about why we would need to prevent getting errors when passing an extra argument? Could you help me elaborate on it, thanks!

Reply:
The reason is that we want to use this format to build single file: `make build library/internet`.

When we run `make build library/internet`, it looks for targets `build` and `library/internet`.
There's no rule for target `library/internet`, so it will throw error.

Here's another 2 possible solution we can try:

1. Use format `make build target=library/internet`.
  In this format, we can get variable `target` = `library/internet`.

2. Find some other
  I need more  time to check is this possible.

Reply:
I got it, thanks!